apps/partners/src/types/data.ts

```
@@ -0,0 +1,45 @@
+import type { InitiativeStatus } from "@cfce/database"
+
+export enum FormMode {
```

Biome (1.9.4)

[error] This enum declaration contains members that are implicitly initialized.

This enum member should be explicitly initialized.

This enum member should be explicitly initialized.

This enum member should be explicitly initialized.

This enum member should be explicitly initialized.

Allowing implicit initializations for enum members can cause bugs if enum declarations are modified over time.
Safe fix: Initialize all enum members.


(lint/style/useEnumInitializers)